src/EditorFeatures/CSharpTest/CodeActions/EncapsulateField/EncapsulateFieldTests.cs

```diff
@@ -1255,5 +1255,89 @@ public int I
 }
 ", options: Option(SimplificationOptions.QualifyFieldAccess, true));
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.EncapsulateField)]
+        public async Task Tuple()
+        {
+            var text = @"
+class foo
+{
+    private (int, string) b[|a|]r;
+
+    void baz()
+    {
+        var q = bar;
```

Comment:
Sure, I'll change. Should the tests above that use the same pattern be updated too?